orderapi/src/test/java/fi/hel/verkkokauppa/order/service/accounting/AccountingSearchServiceTest.java

```diff
@@ -0,0 +1,127 @@
+            assertFalse("Exception: " + e.getMessage(),true);
+        }
+    }
+
+    @Test
+    public void whenGetNotExportedAccountingExportData_thenSuccess() throws Exception {
+        log.info("running whenGetNotExportedAccountingExportData_thenSuccess");
+        createAccountingExportData(10);
+        List<AccountingExportData> resultList;
+        long expectedTotalHits = accountingExportDataCount();
+
+        try{
+            log.info("elasticsearch.search-after-page-size: " + env.getProperty("elasticsearch.search-after-page-size"));
+            resultList = searchServiceToTest.getNotExportedAccountingExportData();
+            log.info("Result list size: "+resultList.size());
+
+            assertFalse("Result list should not be empty",resultList.isEmpty());
+
+            // maximum number of total hits we can receive is 10 000. Cannot verify counts with 10000 or more
+            if(expectedTotalHits < 10000) {
+                assertEquals("Number of results is not the same as number of accountingExportData in ElasticSearch", expectedTotalHits, resultList.size());
+            }
+
+        }
+        catch (Exception e){
+            assertFalse("Exception: " + e.getMessage(),true);
+        }
+    }
+
+    @Test
+    public void whenFindNotAccountedNoHits_thenSuccess() throws Exception {
+        log.info("running whenFindNotAccountedNoHits_thenSuccess");
+        deleteNotAccountedOrders();
+        List<Order> resultList;
+
+        try{
+            log.info("elasticsearch.search-after-page-size: " + env.getProperty("elasticsearch.search-after-page-size"));
+            resultList = searchServiceToTest.findNotAccounted();
+            log.info("Result list size: "+resultList.size());
+
+            assertTrue("Result list should be empty",resultList.isEmpty());
+        }
+        catch (Exception e){
+            assertFalse("Exception: " + e.getMessage(),true);
+        }
+    }
+
+    @Test
+    public void whenFindNotAccounted_thenSuccess() throws Exception {
+        log.info("running whenFindNotAccounted_thenSuccess");
+        createOrder(10);
+        List<Order> resultList;
+        long expectedTotalHits = notAccountedOrderCount();
+
+        try{
+            log.info("elasticsearch.search-after-page-size: " + env.getProperty("elasticsearch.search-after-page-size"));
+            resultList = searchServiceToTest.findNotAccounted();
+            log.info("Result list size: "+resultList.size());
+
+            assertFalse("Result list should not be empty",resultList.isEmpty());
+
+            // maximum number of total hits we can receive is 10 000. Cannot verify counts with 10000 or more
+            if(expectedTotalHits < 10000) {
+                assertEquals("Number of results is not the same as number of accountingExportData in ElasticSearch", expectedTotalHits, resultList.size());
+            }
+
+        }
+        catch (Exception e){
+            assertFalse("Exception: " + e.getMessage(),true);

```

Comment:
https://www.baeldung.com/junit-assert-exception

Check if this should be good?